srcs/incs/CommonDefinitions.hpp

```diff
@@ -4,6 +4,12 @@
 #include <stdint.h>
 #include <string>
 
+#ifdef DBG
+# define dbg() , __FILE__, __FUNCTION__, __LINE__
```

Comment:
This is the first time I see something like this. It means that `dbg()` "expands" to those arguments?

Reply:
yes. It expands to those arguments, and those arguments expand to their actual values, as it already is in main.